test/e2e/framework.go

```diff
@@ -548,6 +623,85 @@ func applyYAML(filename string, ns string) error {
 		return err
 	}
 	outStr, errStr := string(stdout.Bytes()), string(stderr.Bytes())
-	log.Printf("stdout: %s, stderr: %s", outStr, errStr)
+	if outStr != "" {
+		log.Printf("%s", outStr)
+	}
+	if errStr != "" {
+		log.Printf("%s", errStr)
+	}
+	return nil
+}
+
+func runCommand(cmd string) (string, error) {
+	err := wait.Poll(1*time.Second, defaultTimeout, func() (bool, error) {
+		var stdout, stderr bytes.Buffer
+		command := exec.Command("bash", "-c", cmd)
+		log.Printf("Running command %s", cmd)
+		command.Stdout = &stdout
+		command.Stderr = &stderr
+		err := command.Run()
+		if err != nil {
+			return false, nil
+		}
+		outStr, errStr := string(stdout.Bytes()), string(stderr.Bytes())
+		if outStr != "" {
+			log.Printf("%s", outStr)
+		}
+		if errStr != "" {
+			log.Printf("%s", errStr)
+			return false, nil
```

Comment:
If command run succ, is it possible to return (false, nil)?

Reply:
Now, Check by outStr and errStr to decide whether succ or fail